io.github.vinegarhq.Vinegar.yml

```diff
@@ -0,0 +1,233 @@
+  - --socket=x11 # Will be removed or replaced with fallback-x11 once Wine is improved
+  - --share=ipc
+  - --socket=wayland
+  #- --device=dri # Uncomment if --device=all is removed/commented to fix graphics issues
+  - --allow=devel # Necessary for VMProtect to function.
+  - --socket=pulseaudio
+  - --allow=multiarch
+  - --device=all # Necessary for controller support (important on Steam Deck)
+  - --env=WINEDLLPATH=/app/dlls/lib32:/app/dlls/lib
+  - --env=WINEPREFIX=/var/data/wine
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib32/gstreamer-1.0:/app/lib/gstreamer-1.0:/usr/lib/i386-linux-gnu/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0
+
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '22.08'
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    version: '1.4'
+    versions: 22.08;1.4
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d;vulkan/implicit_layer.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+  org.winehq.Wine.gecko:
+    directory: share/wine/gecko
+  org.winehq.Wine.mono:
+    directory: share/wine/mono
+
+x-compat-i386-opts: &compat_i386_opts
+  prepend-pkg-config-path: /app/lib32/pkgconfig:/usr/lib/i386-linux-gnu/pkgconfig
+  # Some autotools-based builds will fail if -L/app/lib32 isn't first
+  ldflags: -L/app/lib32 -Wl,-rpath-link=/app/lib32 -Wl,-z,relro,-z,now -Wl,--as-needed
+  ldflags-override: true
+  append-path: /usr/lib/sdk/toolchain-i386/bin
+  env:
+    CC: ccache i686-unknown-linux-gnu-gcc
+    CXX: ccache i686-unknown-linux-gnu-g++
+  libdir: /app/lib32
+
+separate-locales: false
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /share/man
+  - /bin/function_grep.pl
+  - /bin/widl
+  - /bin/winecpp
+  - /bin/winedump
+  - /bin/wineg++
+  - /bin/winegcc
+  - /bin/winemaker
+  - /bin/wmc
+  - /bin/wrc
+  - /include/wine
+  - /lib/wine/*.def
+  - /lib32/wine/*.def
+
+build-options:
+  append-path: /usr/lib/sdk/mingw-w64/bin
+  env:
+    - PERL5LIB=/app/lib/perl5/
+    - PERL_MM_OPT=INSTALL_BASE=/app
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+  - name: bundle-setup
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/dlls/lib -p
+      - mkdir /app/dlls/lib32 -p
+      - mkdir -p /app/lib32
+      - mkdir -p /app/lib/i386-linux-gnu
+      - install -Dm644 ld.so.conf -t /app/etc/
+    sources:
+      - type: file
+        path: ld.so.conf
+
+  - name: vinegar
+    buildsystem: simple
+    build-commands:
+      - make install GO=$GOROOT/bin/go PREFIX=/app 
+      - install -Dm644 io.github.vinegarhq.Vinegar.metainfo.xml -t /app/share/metainfo
+        # Required for appstream
+      - |
+        icondir=${FLATPAK_DEST}/share/icons/hicolor
+        install -Dm644 catglass.svg ${icondir}/scalable/apps/io.github.vinegarhq.Vinegar.svg 
```

Comment:
this icon isn't square, appstream will reject it.

Reply:
Inkscape reports it as 36x36

Reply:
Additionally, I believe the buildbot passed the icon check stage

Reply:
Must be a github bug then that show it not square.

Reply:
That's odd, the flathub bot is now throwing the same error, but `file io.github.vinegarhq.Vinegar.png` reports

`io.github.vinegarhq.Vinegar.png: PNG image data, 128 x 128, 8-bit/color RGBA, non-interlaced`